### DISCUSSION TITLE
Enabled CarPlay on Audi via SDcard how to restore / remove SDcard

### DISCUSSION THREAD

**Discussion by Scoobdriver** [Mr-MIBonk/M.I.B._More-Incredible-Bash#322] [Q&A]
Firstly Great work on this. 
I enabled CarPlay via More-Incredible-Bash on SDcard and has been working great for a year. 
Time to change car, what happens if I remove the SDcard ? Will the MIB still operate ? 
Am I able to restore ? 

Thanks !

**Comment by petflc** [comment]
Once you done the patches using MIB, you can remove the SD card, everything will remain the same without a SD card.
